lib/ansible/galaxy/role.py

```diff
@@ -379,7 +379,11 @@ def install(self):
                                     if n_part != '..' and not n_part.startswith('~') and '$' not in n_part:
                                         n_final_parts.append(n_part)
                                 member.name = os.path.join(*n_final_parts)
-                                role_tar_file.extract(member, to_native(self.path))
+                                if hasattr(tarfile, 'fully_trusted_filter'):
+                                    # deprecated: description='extract fallback without filter' python_version='3.11'
```

Comment:
Seeing the suggested changes applied, I think it would make more sense to place the deprecation comments above the line without the filter applied (here, and for the other changes), since that's what will eventually be removed.

The comment could be above the conditional, but I think that would make it less obvious what was deprecated.